Microsoft.Toolkit.Services/Services/LinkedIn/LinkedInDataProvider.cs

```diff
@@ -145,16 +145,17 @@ public async Task<bool> LoginAsync()
         /// <summary>
         /// Log user out of LinkedIn.
         /// </summary>
-        public void Logout()
```

Comment:
Readd the method with a deprecated attribute. Call logoutAsync without awaiting.